src/stun_listener.erl

```diff
@@ -212,11 +212,32 @@ udp_recv(Socket, Opts) ->
             erlang:error(Reason)
     end.
 
-format_listener_error(IP, Port, Transport, Opts, Err) ->
+format_listener_error(IP, MinPort, MaxPort, Transport, Opts, Err) ->
     ?LOG_ERROR("Cannot start listener:~n"
                "** IP: ~s~n"
-               "** Port: ~B~n"
+               "** PortRage: ~B-~B~n"
                "** Transport: ~s~n"
                "** Options: ~p~n"
                "** Reason: ~p",
-               [stun_logger:encode_addr(IP), Port, Transport, Opts, Err]).
+               [stun_logger:encode_addr(IP), MinPort, MaxPort, Transport, Opts, Err]).
+
+open_socket(MinPort, MaxPort, OpenSockFunc) ->
+    Count = MaxPort - MinPort,
```

Comment:
Shouldn't be here `Count = MaxPort - MinPort + 1`?

Reply:
Not needed, instead of this I changed pattern matching in few lines below, from `open_socket(_MinPort, _MaxPort, _OpenSockFunc, _Next, 0)` to `open_socket(_MinPort, _MaxPort, _OpenSockFunc, _Next, -1) `